### DISCUSSION TITLE
Experimental operators

### DISCUSSION THREAD

**Discussion by MrSmoke** [morelinq/MoreLINQ#838] [General]
Just a quick one. When do experimental operators get moved into the main project? The `Await()` operator has been around for 3+ years (https://github.com/morelinq/MoreLINQ/pull/205) so it would be nice to have that properly supported with docs and all the jazz

**Comment by atifaziz** [comment]
`Await` is documented as well as most other operators in MoreLINQ. There have also been no issues reported or feedback given so it's hard to gauge if and how thoroughly it's being used in the wild. You're the first to inquire about its status! There's really no criterion for when something is moved out of the `Experimental` namespace; one usually hopes to hear back soon after publishing an experimental piece to iron out any design/implementation kinks before considering it golden. What's your concern? That it might disappear from a future release? Have you been using it thoroughly in production applications or something you're planning to do?
